app/grants/clr.py

```diff
@@ -292,7 +292,12 @@ def calculate_clr(curr_agg, trust_dict, pair_totals, curr_agg_sqrts, v_threshold
                 u_k2 = k2 if k2 > k1 else k1
                 # never pair against matching contribs
                 if k2 != k1 and pair_totals[u_k1][u_k2] > 0:
-                    tot += (sqrts[u_k1] * sqrts[u_k2]) / (pair_totals[u_k1][u_k2] / (v_threshold * float(max(trust_dict[u_k1], trust_dict[u_k2]))) + 1)
+                    try:
+                        tot += (sqrts[u_k1] * sqrts[u_k2]) / (pair_totals[u_k1][u_k2] / (v_threshold * float(max(trust_dict[u_k1], trust_dict[u_k2]))) + 1)
+                    except Exception as e:
+                        print(sqrts[u_k1], sqrts[u_k2], pair_totals[u_k1][u_k2], v_threshold, trust_dict[u_k1], trust_dict[u_k2] )
+                        print(f"error: calculate_clr {e}")
+                        continue
```

Comment:
this isn't required as it's the end of the loop but keeping it nevertheless just to be safe